osgtest/tests/test_54_gratia.py

```diff
@@ -411,10 +411,9 @@ def test_17_copy_condor_logs(self):
     #This test executes condor_meter
     def test_18_execute_condor_meter(self):
         core.state['gratia.condor-meter-running'] = False
-        core.skip_ok_unless_installed('gratia-probe-condor', 'gratia-service', 'globus-gram-job-manager-condor',
-                                      'globus-gram-client-tools', 'globus-proxy-utils')
+        core.skip_ok_unless_installed('gratia-probe-condor', 'gratia-service', 'osg-ce')
```

Comment:
That was my original thought. But from talking with @osg-cat, I thought the metapackage osg-ce is used to install either of them. So it would be enough to check for it.